### ISSUE TITLE
How to configure GdSceneSynchronizer in case of two connections?

### ISSUE THREAD

**Issue by Malkverbena** [GameNetworking/network_synchronizer#120]
If I have two connections in the same scene, how should I configure GdSceneSynchronizer?
Would I have to have two singletons?

**Comment by AndreaCatania** [comment]
Yes, or a single singleton that under the hood instantiates two (or more) different `GdSceneSynchronizer`.
In such a case the API would look like:
```
NetSync.sync_layer[0].register_variable(...)
```

**Comment by Malkverbena** [comment]
I guess that multiplayer of GdSceneSynchronizer also must be set, correct?

**Comment by AndreaCatania** [comment]
Yes, each GdSceneSynchronizer must have it's own (or shared) MultiplayerAPI set as usual.
